src/wallet/coinselection.cpp

```diff
@@ -298,3 +298,27 @@ bool KnapsackSolver(const CAmount& nTargetValue, std::vector<CInputCoin>& vCoins
 
     return true;
 }
+
+/*
+ * Randomly selects coins until the target value is exceeded. Uses effective values
+ */
+bool SingleRandomDraw(const CAmount& target_value, std::vector<CInputCoin>& utxo_pool, std::set<CInputCoin>& out_set, CAmount& value_ret, CAmount not_input_fees)
```

Comment:
`non_input_fees`? It's not "not fees", rather "non-input". Easier for me to read.

Reply:
Done